v3/how-to-guides/02-pallet-design/c-randomness/index.mdx

```diff
@@ -80,7 +80,7 @@ Use a nonce to serve as a subject for the `frame_support::traits::Randomness::ra
 
       // Random value.
       let nonce = Self::get_and_increment_nonce();
-      let randomValue = T::MyRandomness::random(&nonce);
+      let (randomValue, _) = T::MyRandomness::random(&nonce);
```

Comment:
Hi @dzmitry-lahoda - why is this necessary ? 🤔

Reply:
why i split random value out of tuple?

Reply:
Yes. I'm assuming it's because it takes `<Hash, BlockNumber>` ? But I'm not sure it's necessary.